pkgs/development/python-modules/pyxdg/default.nix

```diff
@@ -6,24 +6,16 @@
 
 buildPythonPackage rec {
   pname = "pyxdg";
-  version = "0.26";
+  version = "0.27";
 
   src = fetchPypi {
```

Comment:
Use fetchgit to fetch from https://gitlab.freedesktop.org/xdg/pyxdg, so we can have tests.

Reply:
I used fetchFromGitLab, done

Reply:
Of note, this fixes one test error